### ISSUE TITLE
Tanto Wagmi waypoint() CreateConnectorFn creates two connectors

### ISSUE THREAD

**Issue by seuchenhund** [skymavis/tanto-kit#18]
I noticed that the tanto-wagmi example creates 3 buttons: Connect to Ronin Wallet, Connect to Ronin Waypoint, Connect to Ronin Wallet.

https://github.com/skymavis/tanto-kit/blob/0dcbfd179b053921cb5a680cd8acb46a0ede4ca0/packages/wagmi/README.md?plain=1#L33

https://github.com/skymavis/tanto-kit/blob/0dcbfd179b053921cb5a680cd8acb46a0ede4ca0/packages/wagmi/README.md?plain=1#L64-L68

Looks like the CreateConnectorFn `waypoint()` on its own creates a Waypoint connector AND a Ronin Wallet connector. Not sure if that's on purpose.

**Comment by HoangNguyen17193** [comment]
@greyy-nguyen @npnam93 Please take a look on this issue

**Comment by greyy-nguyen** [comment]
Can you provide images or details about how you got this error? Since I couldn't replicate this issue.

**Comment by seuchenhund** [comment]
@greyy-nguyen after looking into it a bit more, I found the root cause of the double Ronin Wallet connectors.

You're right and the `waypoint()` as well as the `roninWallet()` `createConnectorFn` only returns a single connector.

The [auto discovery of injected providers](https://wagmi.sh/react/api/createConfig#multiinjectedproviderdiscovery) was adding a second Ronin Wallet connector.

So I just set `multiInjectedProviderDiscovery` to `false` to solve this issue, as my app doesn't need to find other wallets anyway.

```
export const config = createConfig({
  chains: [ronin, saigon],
  multiInjectedProviderDiscovery: false,
  transports: {
      [ronin.id]: http(),
      [saigon.id]: http(),
  },
  connectors: [waypoint() as CreateConnectorFn, roninWallet() as CreateConnectorFn], 
})
```

Sorry for the false alert. I'm not sure if this is something we need to note in the code example. I'll let you guys decide.
